marvin.py

```diff
@@ -31,37 +31,38 @@ def get_page_title_from_url(page_url: str):
 # update. Error handlers also receive the raised TelegramError object in error.
 def start(bot, update):
     """Send a message when the command /start is issued."""
-    update.message.reply_text('Hi!')
-
-
-def help(bot, update):
-    """Send a message when the command /help is issued."""
-    update.message.reply_text('Help!')
+    update.message.reply_text('In un gruppo, rispondi ad un link con il comando /postalink')
 
 
-def postalink(bot, update):
+def postalink(subreddit, bot, update):
     # print("Reply from:" + str(update.message.reply_to_message.text))
-    message_entities_dict = update.message.reply_to_message.parse_entities([MessageEntity.URL])
-    print("Size of entities dict:" + str(len(message_entities_dict)))
-    # print("Author of the post: " + update.message.from_user.username)
-    print("Author of the post: " + update.message.from_user.name)
-    if len(message_entities_dict) == 1:
-        link_to_post = str(update.message.reply_to_message.parse_entity(next(iter(message_entities_dict))))
-        print("Link to post:" + link_to_post)
-        link_page_title = get_page_title_from_url(link_to_post)
-        print("Website title:" + link_page_title)
-        # submit to reddit:
-        global subreddit
-        # Create the post title
-        title = "[Post from telegram by:" + update.message.from_user.name + "]" + link_page_title
-        submission = subreddit.submit(title, url=link_to_post)
-        print("Link to created post:" + str(submission.shortlink))
-        update.message.reply_text("Post creato:" + str(submission.shortlink))
-    else:
-        update.message.reply_text("Non posso postare quel contenuto...")
-
-
-def error(bot, update, error):
+    if not update.message.reply_to_message:
```

Comment:
Questo è già controllato da `Filters.reply` sotto quando si crea l'handler, in che caso entra in quest'if?

Reply:
Teoricamente in nessuno, ma non costa veramente nulla e difende da errori di codice (o di Telegram).